.php_cs.dist

```diff
@@ -8,200 +8,198 @@ For the full copyright and license information, please view the LICENSE
-            'return_type_declaration' => ['space_before' => 'none'],
-            'self_accessor' => true,
-            'semicolon_after_instruction' => true,
-            'set_type_to_cast' => true,
-            'short_scalar_cast' => true,
-            'simplified_null_return' => false,
-            'single_blank_line_at_eof' => true,
-            'single_import_per_statement' => true,
-            'single_line_after_imports' => true,
-            'single_quote' => true,
-            'standardize_not_equals' => true,
-            'ternary_to_null_coalescing' => true,
-            'trailing_comma_in_multiline_array' => true,
-            'trim_array_spaces' => true,
-            'unary_operator_spaces' => true,
-            'visibility_required' => [
-                'elements' => [
-                    'const',
-                    'method',
-                    'property',
-                ],
+        ],
+        'ordered_imports' => true,
+        'ordered_interfaces' => [
+            'direction' => 'ascend',
+            'order' => 'alpha',
+        ],
+        'phpdoc_add_missing_param_annotation' => true,
+        'phpdoc_align' => true,
+        'phpdoc_annotation_without_dot' => true,
+        'phpdoc_indent' => true,
+        'phpdoc_no_access' => true,
+        'phpdoc_no_empty_return' => true,
+        'phpdoc_no_package' => true,
+        'phpdoc_order' => true,
+        'phpdoc_return_self_reference' => true,
+        'phpdoc_scalar' => true,
+        'phpdoc_separation' => true,
+        'phpdoc_single_line_var_spacing' => true,
+        'phpdoc_to_comment' => true,
+        'phpdoc_trim' => true,
+        'phpdoc_trim_consecutive_blank_line_separation' => true,
+        'phpdoc_types' => ['groups' => ['simple', 'meta']],
+        'phpdoc_types_order' => true,
+        'phpdoc_var_without_name' => true,
+        'pow_to_exponentiation' => true,
+        'protected_to_private' => true,
+        'return_assignment' => true,
+        'return_type_declaration' => ['space_before' => 'none'],
+        'self_accessor' => true,
+        'semicolon_after_instruction' => true,
+        'set_type_to_cast' => true,
+        'short_scalar_cast' => true,
+        'simplified_null_return' => false,
+        'single_blank_line_at_eof' => true,
+        'single_import_per_statement' => true,
+        'single_line_after_imports' => true,
+        'single_quote' => true,
+        'standardize_not_equals' => true,
+        'ternary_to_null_coalescing' => true,
+        'trailing_comma_in_multiline_array' => true,
+        'trim_array_spaces' => true,
+        'unary_operator_spaces' => true,
+        'visibility_required' => [
+            'elements' => [
+                'const',
+                'method',
+                'property',
             ],
-            'void_return' => true,
-            'whitespace_after_comma_in_array' => true,
-        ]
-    )
-    ->setFinder(
-        PhpCsFixer\Finder::create()
-        ->files()
```

Comment:
Introducing an explaining variable allows us to get rid of the a bit awkward indentation here.